configure.ac

```diff
@@ -1880,6 +1880,48 @@ then
 fi
 
 
+# Maxim Integrated MAXQ10XX
+ENABLED_MAXQ10XX="no"
+maxqpartnumber=""
+AC_ARG_WITH([maxq10xx],
+    [AS_HELP_STRING([--with-maxq10xx=PART],[MAXQ10XX PART Number])],
+    [
+        AC_MSG_CHECKING([for maxq10xx])
+
+        # Read the part number
+        maxqpartnumber=$withval
+
+        if test "$maxqpartnumber" = "MAXQ1065"; then
+            LIB_STATIC_ADD="$LIB_STATIC_ADD lib/libmaxq1065_api.a"
+            AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_MAXQ1065"
+            ENABLED_MAXQ10XX="yes"
+        elif test "$maxqpartnumber" = "MAXQ108x"; then
+            LIB_STATIC_ADD="$LIB_STATIC_ADD lib/libmaxq108x_api.a"
+            AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_MAXQ108X"
+            ENABLED_MAXQ10XX="yes"
+        else
+            AC_MSG_ERROR([need a valid MAXQ part number])
+        fi
+
+        case $host_os in
+        mingw* | cygwin* | msys* | pw32* | os2* | cegcc*)
+                ENABLED_MAXQ10XX="yes"
+                LIB_ADD="$LIB_ADD -lcrypt32"
+                LIB_STATIC_ADD="lib/LibFT4222-64.dll lib/ftd2xx-64.dll $LIB_STATIC_ADD"
```

Comment:
Is this pulling in an FTDI driver? Seems odd to require this in our code. It should be in the end application not in the library.

Reply:
This is required for examples, not library.

Reply:
I'm wondering if this could be handled in the  `./configure`  step via the `LIBS` and/or `LDFLAGS` environment variables.